bin/trigger_azure_pipelines.py

```diff
@@ -74,7 +78,8 @@ def main() -> None:
         organization = os.environ["ADO_ORGANIZATION"]
         project = os.environ["ADO_PROJECT"]
         pipeline_id = os.environ["ADO_AZUREAUTH_LINUX_PIPELINE_ID"]
-
+        version = os.environ["VERSION"]

```

Comment:
Are the env var names `VERSION` and `GITHUB_SHA` ones we control through Github secrets? I'm wondering if we can make these names slightly more descriptive, because on their own, they could mean a lot of different things.

Reply:
`GITHUB_SHA` environment variable is available by default for workflows. https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables 

`VERSION` is the release version. We need to set this as an environment variable in the release workflow. We get this value currently from `${{ github.event.inputs.version }}`

Reply:
How about updating `VERSION` to `RELEASE_VERSION`?

Reply:
I figured `VERSION` was implied because it's a GitHub release, but if it's confusing we can give it more context. Perhaps `AZUREAUTH_VERSION`? When I've created releases via the CLI (which uses the release workflow) I've run
```
gh workflow run Release --field version=<whatever> --field prerelease=true
```
, but I think it makes just as much sense to do
```
gh workflow run Release --field azureauth_version=<whatever> --field prerelease=true
```

Reply:
@reillysiemens when you try
```
gh workflow run Release --field version=<whatever> --field prerelease=true
```
does `${{ github.event.inputs.version }}` holds the <whatever> ?

Reply:
Ah that makes more sense, I am fine leaving it as `VERSION` - because we won't have an arbitrary `VERSION` set outside that context of the input field. Coming from the release workflow input field makes sense. Thank you for clarifying!